server/Gemfile

```diff
@@ -33,9 +43,13 @@ group :development, :test do
 
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'rspec-rails', '~> 3.9'
+  # Adds support for Capybara system testing and selenium driver
+  gem 'pry-rescue'
+  gem 'pry-stack_explorer'
+  gem 'capybara', '~> 3.31'
+  gem 'selenium-webdriver'
```

Comment:
All of these gems  can be added as we go along, right? By adding them, we prolong the install cycle in Travis CI by quite a bit, perhaps we can add them in when we add the first Capybara test?